### PR TITLE
Add version handling for the dev packages in Composer when creating the release

### LINKED ISSUE #14782: Cannot activate 2fa
### Describe the bug

Trying to activate two factor authentication, but encountering this error:

[Thu Dec 13 21:12:23.455404 2018] [php7:error] [pid 18885] [client x.x.x.x:30233] PHP Fatal error:  Uncaught Error: Call to undefined method PragmaRX\\Google2FA\\Google2FA::getQRCodeInline() in /usr/local/www/web/dir/html/admin/libraries/classes/Plugins/TwoFactor/Application.php:96\nStack trace:\n#0 /usr/local/www/web/dir//html/admin/libraries/classes/TwoFactor.php(219): PhpMyAdmin\\Plugins\\TwoFactor\\Application->setup()\n#1 /usr/local/www/web/dir//html/admin/prefs_twofactor.php(34): PhpMyAdmin\\TwoFactor->setup()\n#2 {main}\n  thrown in /usr/local/www/web/dir//html/admin/libraries/classes/Plugins/TwoFactor/Application.php on line 96

Running on amazon linux with php 7.1 from amazon repo.  

Package php71-gd-7.1.23-1.34.amzn1.x86_64 installed.

### To Reproduce

1. Install brand new phpmyadmin 4.8.4
2. Click settings -> Two-Factor Authentication
3. Select 'Authentication Application (2FA)'
4. Click "Configure two factor authentication"
5. Error message: Error in processing request 
Error code: 500
Error text: Internal Server Error (rejected)
It seems that the connection to server has been lost. Please check your network connectivity and server status.

Error message shown above is in error log.

### Expected behavior

Two factor authentication setup.

### Server configuration

- Operating system: Amazon Linux 4.14.77-70.82.amzn1.x86_64
- Web server: Apache/2.4.34 (Amazon)
- Database version: 5.7.24 - MySQL Community Server (GPL) 
- PHP version: 7.1.23 
- phpMyAdmin version: 4.8.4

### Client configuration

- Browser: Firefox 64
- Operating system: Win 7

### PR DESCRIPTION
**Pull request by ibennetch**
This parses the required dependency versions from composer.json and installs them manually as required packages.

Fixes #14782

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
LGTM :)

**Review by farzadarab — COMMENTED** [review]
i can not sigh in to my iphon accunt

File: scripts/create-release.sh [hunk 1]
```diff
@@ -235,10 +235,19 @@ if [ ! -d libraries/tcpdf ] ; then
...
+            PACKAGES_VERSIONS="$PACKAGES_VERSIONS $PACKAGES:`awk "/require-dev/ {printline = 1; print; next } printline" composer.json | grep "$PACKAGES" | awk -F [\\"] '{print $4}'`"
+        done
+        composer require --update-no-dev $PACKAGES_VERSIONS
     else
-        composer require --update-no-dev tecnickcom/tcpdf pragmarx/google2fa-qrcode samyoul/u2f-php-server
+        for PACKAGES in tecnickcom/tcpdf pragmarx/google2fa-qrcode samyoul/u2f-php-server
+        do
+            PACKAGES_VERSIONS="$PACKAGES_VERSIONS $PACKAGES:`awk "/require-dev/ {printline = 1; print; next } printline" composer.json | grep "$PACKAGES" | awk -F [\\"] '{print $4}'`"
+        done
+        composer require --update-no-dev $PACKAGES_VERSIONS
```
[the remark this answers is not in the document]
Comment (ibennetch) [reply scripts/create-release.sh]: Great idea.

**Comment by ibennetch** [comment]
I've updated this a bit and think it's better now.

**Comment by ibennetch** [comment]
@farzadarab are you talking about phpMyAdmin? We don't have a native iPhone app so I'm a bit confused.

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**